docs/getting-started/installation.md

```diff
@@ -17,37 +18,162 @@ This can be done with the following command:
     
 `/usr/sbin/softwareupdate --install-rosetta --agree-to-license`
 
-## Virtual Environment (Optional)
+
+```python eval
+rx.alert(
+    rx.alert_icon(),
+    rx.box(
+        rx.alert_title("Error ", rx.code("command not found: reflex"),),
+        rx.alert_description(
+            "If you install Reflex with no virtual environment and get this error it means your PATH cannot find the reflex package. A virtual environment should solve this problem.",
+        ),
+    ),
+    status="warning",
+)
+```
+
+
+## Virtual Environment (Recommended)
 
 We recommend creating a virtual environment for your project.
 
 Below are some tools you can use to manage environments:
 
 - [venv]({constants.VENV_URL})
 - [poetry]({constants.POETRY_URL})
+- [conda]({constants.CONDA_URL})
+
+```python eval
+rx.box(height=8)
+```
+
+
+
+## Installing on macOS/Linux
 
-## Installing
+Reflex's recommended package manager and environment manager for macOS and Linux are [pip]({constants.PIP_URL}) and [venv]({constants.VENV_URL}), respectively. `venv` is a part of [The Python Standard Library]({constants.PYTHON_STANDARD_LIBRARY}) and comes bundled with your installation of Python. See instructions on how to install and use `pip` below.
 
-Reflex is available as a [pip](constants.PIP_URL) package.
+
+### Install pip
+
+Install `pip`. More details about installing pip can be found in [pip's documentation]({constants.PIP_DOCS}).
+
+On a macOS:
 
 ```bash
-pip install reflex
+python -m ensurepip --upgrade
 ```
 
-## Create a Project
+```python eval
+rx.alert(
+    rx.alert_icon(),
+    rx.box(
+        rx.alert_title("Error ", rx.code("command not found: python"),),
+        rx.alert_description(
+            "On mac sometimes you need to write python3 instead of python",
+        ),
+    ),
+    status="warning",
+)
+```
 
-Installing Reflex also installs the `reflex` command line tool.
+```python eval
+rx.box(height=2)
+```
 
-Test that the install was successful by creating a new project. Replace `{app_name}` with your project name:
+On Ubuntu with Python 3:
 
 ```bash
-$ mkdir {app_name}
-$ cd {app_name}
-$ reflex init
+sudo apt-get install python3-pip
```

Comment:
are we sure this is sufficient?

In my testing, I've had to install `python3-venv` to get the venv module (and venvs automatically come with `pip`). The main reason we would want `python3-pip` is because it also depends on compilers and headers needed to compile c extensions.

Reply:
changed